flybywiresim/a32nx
# refactor(mcdu): improved departure/arrival pages

Fixes #[issue_no]

## Summary of Changes


Fixed an issue with being unable to select certain enroute transitions

* Minor overhaul of arrivals/departures page for proper pagination for all fields
* Improved cosmetic appearance to be closer to true-to-life
* Modified routing to be closer to true-to-life behaviour

## Screenshots (if necessary)



## References

https://www.youtube.com/watch?v=twwUjHXdNVU




## Additional context



Discord username (if different from GitHub):

## Testing instructions


1. Spawn and enter the following routes in (via sim-brief or manual entry)
 KLAS/26R RADYR2 BLAQQ DCT SHTNR ANJLL4 KLAX/24R
 KFLL/28R REGAE1 PERMT AR16 ILM SBY J79 VILLS DPK DPK3 KBDL/24
ESGG/21 TOPLA3J TOPLA DCT PENOR N133 DOMAG T240 OGDAV OGDAV4T EPGD/29

1. You are able to successfully enter these routes with the departures and arrival pages in the MCDU.

1. MCDU does not break on these pages when buttons are mashed/pressed in an odd order

1. Selecting, removing, then selecting a departure/arrival works as expected.


## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created, built, and uploaded.

1. Make sure you are sig1ned in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, click on the bottom **PR** tab
1. Click on the **A32NX** download link at the bottom of the page
